source/mqtt/Mqtt5Client.cpp

```diff
@@ -18,71 +18,6 @@ namespace Aws
 {
     namespace Crt
     {
-
```

Comment:
Moved to Mqtt5Types.h so we can shared the ScopedRWLock between Mqtt5Client.cpp and Mqtt5Listener.cpp.